packer/build_deb_image.sh

```diff
@@ -249,8 +255,10 @@ elif [ "$TARGET" = "azure" ]; then
     PACKER_ARGS+=(-var client_secret="$AZURE_CLIENT_SECRET")
     PACKER_ARGS+=(-var tenant_id="$AZURE_TENANT_ID")
     PACKER_ARGS+=(-var subscription_id="$AZURE_SUBSCRIPTION_ID")
-    PACKER_ARGS+=(-var scylla_build_id="$BUILD_ID")
+fi
 
+if $DEBUG ; then
+  PACKER_ARGS+=(-var image_prefix="releng-testing-")
```

Comment:
Too specific to releng... What if somebody else is building the image for debugging purposes?
Better to set a prefix name as "image-debug-"

Reply:
Changed